### DISCUSSION TITLE
When used parameter “ --test_transforms ” ，about Image depth。

### DISCUSSION THREAD

**Discussion by XJTU-JP** [NVlabs/instant-ngp#354] [Q&A]
hello，when i used parameter "--test_transforms" to compute PSNR，I encountered a problem about the image depth。i use images that is 24 bit in depth，but the ngp use 32 bit to compute PSNR。As shown in the follow。Do I need to unify all training and test data into 32 bits？Thank you for your answer
`image shape =  (768, 1024, 4)`
`Traceback (most recent call last):`
`File "D:\test2\instant-ngp\scripts\run.py", line 276, in <module>`
`diffimg = np.absolute(image - ref_image)`
`ValueError: operands could not be broadcast together with shapes (768,1024,4) (768,1024,3)`

**Comment by Tom94** [comment]
Hi there, I'm afraid that's the easiest solution, yes. The script could also be adapted to append an `alpha==1` channel to any loaded RGB reference image.
